### PR TITLE
Add notificationPublisher/settings resource

### PR DESCRIPTION
**Pull request by PingDavidR**
- closes PDI-1387

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# v0.7.0 (Unreleased)
+### DEPRECATED
```
Comment (henryrecker-pingidentity) [inline CHANGELOG.md:2]: imo we don't need the empty section headers

File: internal/resource/config/notificationpublishers/settings/notification_publishers_settings_resource.go [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+	}
+	apiReadNotificationPublishersSettings, httpResp, err := r.apiClient.NotificationPublishersAPI.GetNotificationPublishersSettings(config.ProviderBasicAuthContext(ctx, r.providerConfig)).Execute()
+
+	if err != nil {
+		if httpResp != nil && httpResp.StatusCode == 404 {
+			config.ReportHttpErrorAsWarning(ctx, &resp.Diagnostics, "An error occurred while getting the Notification Publishers Settings", err, httpResp)
+			resp.State.RemoveResource(ctx)
+		} else {
+			config.ReportHttpError(ctx, &resp.Diagnostics, "An error occurred while getting the Notification Publishers Settings", err, httpResp)
+		}
```
Comment (henryrecker-pingidentity) [inline internal/resource/config/notificationpublishers/settings/notification_publishers_settings_resource.go:140]: Need a return after this line

File: internal/resource/config/notificationpublishers/settings/notification_publishers_settings_resource.go [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+func (r *notificationPublishersSettingsResource) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
+	var err error
+	var plan notificationPublishersSettingsResourceModel
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	//	updateNotificationPublishersSettings := r.apiClient.NotificationPublishersAPI.UpdateNotificationPublishersSettings(config.ProviderBasicAuthContext(ctx, r.providerConfig), plan.VALUE.ValueString())
```
Comment (henryrecker-pingidentity) [inline internal/resource/config/notificationpublishers/settings/notification_publishers_settings_resource.go:166]: Remove commented line

**Review by henryrecker-pingidentity — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (PingDavidR) [reply CHANGELOG.md:2]: I debated about this with myself.  Easy enough to add back.  Done.

File: internal/resource/config/notificationpublishers/settings/notification_publishers_settings_resource.go (hunk 2 above)
Comment (PingDavidR) [reply internal/resource/config/notificationpublishers/settings/notification_publishers_settings_resource.go:140]: done

**Review by henryrecker-pingidentity — APPROVED** [review]

**Pull request merged**